pkgs/top-level/all-packages.nix

```diff
@@ -29399,7 +29399,9 @@ with pkgs;
 
   lmodern = callPackage ../data/fonts/lmodern { };
 
-  logiops = callPackage ../misc/drivers/logiops { };
+  logiops_0_2_3 = callPackage ../misc/drivers/logiops/0_2_3.nix { };
+  logiops_0_3_3 = callPackage ../misc/drivers/logiops/0_3_3.nix { };
+  logiops = logiops_0_3_3;
```

Comment:
0.2.3 is more of an exception so `logiops = callPackage … 0_3_3.nix` is fine & encouraged here.